core/src/main/java/org/bitcoinj/core/Coin.java

```diff
@@ -118,6 +118,46 @@ public static Coin valueOf(final int coins, final int cents) {
         return coin;
     }
 
+    /**
+     * Convert a decimal amount of BTC into satoshis.
+     *
+     * @param coins number of coins
+     * @return number of satoshis
+     */
+    public static long btcToSatoshi(BigDecimal coins) {
+        return coins.movePointRight(SMALLEST_UNIT_EXPONENT).longValueExact();
+    }
+
+    /**
+     * Convert an amount in satoshis to an amount in BTC.
+     *
+     * @param satoshis number of satoshis
+     * @return number of bitcoins (in BTC)
+     */
+    public static BigDecimal satoshiToBtc(long satoshis) {
+        return new BigDecimal(satoshis).movePointLeft(SMALLEST_UNIT_EXPONENT);
+    }
+
+    /**
+     * Convert a decimal amount of BTC into satoshis.
```

Comment:
Shouldn't this read "…to a Coin", similar to the below method?

Reply:
fixed in 165341fbf4a027f9375561964b52a0e88f9073d1 (and made similar changes to other static factory methods)